axiom/nr_segment_datastore.c

```diff
@@ -208,11 +216,14 @@ bool nr_segment_datastore_end(nr_segment_t** segment_ptr,
    * The allWeb and allOther rollup metrics are created at the end of the
    * transaction since the background status may change.
    */
-  scoped_metric
-      = create_metrics(segment, duration, datastore_string, collection,
-                       operation, &datastore, params->instance);
-
-  nr_segment_set_name(segment, scoped_metric);
+  if (!params->instance_only) {
+    scoped_metric
+        = create_metrics(segment, duration, datastore_string, collection,
+                         operation, &datastore, params->instance);
+    nr_segment_set_name(segment, scoped_metric);
```

Comment:
Should the segment get the name in the case only instance metrics get created as well? Is the segment name important in such case at all?

Reply:
This comment is obsolete as of 3ed5bc54bb0d90bc792ed32cd650cfb87c7de972